txhelpers/staketree.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/decred/dcrd/chaincfg"
 	"github.com/decred/dcrd/chaincfg/chainhash"
 	"github.com/decred/dcrd/database"
-	_ "github.com/decred/dcrd/database/ffldb"
+	//_ "github.com/decred/dcrd/database/ffldb"
```

Comment:
this might have been needed.  I think there is an `init()` method that gets called with this anonymous import